test/features/pagecontent/revisions.js

```diff
@@ -8,36 +8,37 @@ var preq   = require('preq');
 var server = require('../../utils/server.js');
 var pagingToken = '';
 
-describe('revision requests', function() {
-
-    var revOk = 642497713;
-    var revDeleted = 645504917;
-    var revRedirect = 591082967;
-    var pageName = 'User:GWicke%2fDate';
-    var pageLastRev = 653530930;
-
-    this.timeout(20000);
-
-    before(function () { return server.start(); });
+function generateTests(options) {
+
+    var apiRequestTemplate = server.config.conf.templates['wmf-sys-1.0.0']
+                .paths['/{module:action}']['x-modules'][0]
+                .options.apiRequest;
```

Comment:
Since this string is long already, I think it would be slightly more readable if, starting with `wmf-sys-1.0.0`, all of the properties were in *hash notation* (*`['prop']`*). That could provide a visual aid in identifying the complete path sought.